src/main/bash/sdkman-path-helpers.sh

```diff
@@ -82,7 +82,7 @@ function __sdkman_link_candidate_version {
 
 	# Change the 'current' symlink for the candidate, hence affecting all shells.
 	if [[ -h "${SDKMAN_CANDIDATES_DIR}/${candidate}/current" || -d "${SDKMAN_CANDIDATES_DIR}/${candidate}/current" ]]; then
-		rm -f "${SDKMAN_CANDIDATES_DIR}/${candidate}/current"
+		rm -rf "${SDKMAN_CANDIDATES_DIR}/${candidate}/current"
```

Comment:
`-r` does not change anything here, because `current` is a symlink.

Reply:
My terminal is MSYS2 on the windows 7, and creating a directory instead of symlink, so '-r' is needed.

Reply:
Sure, but that's a bug very specific to Windows, and forcefully removing a directory that msys has created doesn't solve anything. In fact, it would thus also delete a previously installed candidate! That is an unwanted side effect.

What we really want is msys to handle symlinks in the same way that all other Unix platforms do. Is there perhaps some kind of flag or setting we could use here for cygwin/msys?

Reply:
MSYS2 will creating really directory after executing 'ln -s xx xx', you can see this [issue](https://sourceforge.net/p/msys2/tickets/41/), so as your saying, it might specific deal with thing for MSYS2 on windows